variables.tf

```
@@ -712,3 +712,12 @@ variable "create_route53_aaaa_record" {
   type        = bool
   default     = false
 }
+
+variable "runtime_platform" {
+  description = "Configuration block for runtime_platform that containers in your task may use."
+  type = object({
```

Since these are both [optional](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecs_task_definition#runtime_platform), we don't want to lock users into providing the full schema. So lets just use
```suggestion
  type = any
```